experiments/bc_benchmark.sh

```diff
@@ -11,7 +11,8 @@ source experiments/common.sh
 
 ENVS=(seals_cartpole)
 SEEDS=(0 1 2)
-OUTPUT_DIR="output/bc_benchmark/${TIMESTAMP}"
+# To prevent race conditions, we use a different output dir for each process id.
+OUTPUT_DIR="output/bc_benchmark/${TIMESTAMP}/$BASHPID"
```

Comment:
Preference for this being the same directory so we don't create lots of directories with a single subdirectory.

Also suggest using `${}` for consistency with surrounding code although in this situation it doesn't matter.

```suggestion
OUTPUT_DIR="output/bc_benchmark/${TIMESTAMP}-${BASHPID}"
```